.github/workflows/code-coverage.yml

```diff
@@ -39,25 +37,72 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-composer-
 
+      # As suggested here https://github.com/codeclimate/test-reporter/issues/406#issuecomment-578483422
+      - name: Set ENV for CodeClimate (pull_request)
+        if: github.event_name == 'pull_request'
+        run: |
+          echo $GITHUB_HEAD_REF >> $GIT_BRANCH
+          echo "$(git rev-parse origin/$GITHUB_HEAD_REF)" >> $GIT_COMMIT_SHA
+
+      - name: Set ENV for CodeClimate (push)
+        if: github.event_name == 'push'
```

Comment:
I note we don't run on push (line 2 above only specifies `pull_request`), so this step seems unneeded (and line 42 redundant). OTOH, it doesn't hurt anything if you want to keep it around.

Reply:
it will once https://github.com/Automattic/jetpack/pull/17997 is merged